lib/std/stropt.nim

```diff
@@ -0,0 +1,63 @@
+#
+#
+#           The Nim Compiler
+#        (c) Copyright 2020 Nim Contributors
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+proc strip*(s: var string, leading = true, trailing = true,
+                   chars: set[char] = {' ', '\t', '\v', '\r', '\l', '\f'}) =
```

Comment:
This should have ` first = 0.Natural; last: 0.Natural; `, what if I know that my `string` has 
48723409873289213 trailing and leading white spaces and 1 non-whitespace char thats useful?.
Nothing changes if you do not know.

Reply:
=> https://github.com/nim-lang/Nim/pull/16280#issuecomment-749982829